doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,281 @@
+# <a name="top"></a>This document describes all the headers used by Orion, as defined in src/lib/rest/HttpHeaders.h file.
```

Comment:
This doesn't look very well...

![imagen](https://user-images.githubusercontent.com/1534240/73446306-cc306f80-435c-11ea-95b4-4617a96e4110.png)

Suggestion:

* Use as title "Orion HTTP headers"
* After the title add the following paragraph (note the filename in bold text style):

> This document describes all the headers used by Orion, as defined in **src/lib/rest/HttpHeaders.h**

Reply:
Fixed in 512f9cd